tests/gateway/test_client.py

```diff
@@ -24,7 +24,7 @@ def basic_config_dict():
                         "openai_api_key": "mykey",
                         "openai_api_base": "https://api.openai.com/v1",
                         "openai_api_version": "2023-05-10",
-                        "openai_api_type": "openai/v1/chat/completions",
```

Comment:
This was invalid. Validation logic introduced in this PR caused this test case to fail & similar ones below